src/CookieJar.php

```diff
@@ -12,7 +12,7 @@ final class CookieJar implements \Countable, \IteratorAggregate
     /**
      * @var \SplObjectStorage
      */
-    protected $cookies;
+    private $cookies;
```

Comment:
Fixing coding style via `PHP-CS-Fixer`.

Reply:
The coding style issue is as follows:

````
PHP CS Fixer 2.17.1 Desert Beast by Fabien Potencier and Dariusz Ruminski (5198b73)
Loaded config default from "/github/workspace/.php_cs.dist".
   1) src/CookieJar.php
      ---------- begin diff ----------
--- Original
+++ New
@@ -12,7 +12,7 @@
     /**
      * @var \SplObjectStorage
      */
-    protected $cookies;
+    private $cookies;
 
     public function __construct()
     {
@@ -92,7 +92,7 @@
      *
      * @return Cookie[]
      */
-    protected function findMatchingCookies(callable $match)
+    private function findMatchingCookies(callable $match)
     {
         $cookies = [];
 

      ----------- end diff -----------


Checked all files in 0.659 seconds, 12.000 MB memory used
````

Reply:
i was worried for a bit, but then noticed that the class is final, so indeed having protected properties/methods makes no sense.